src/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -507,17 +507,35 @@ internal enum GcRestrictedCalloutKind
         [RuntimeImport(RuntimeLibrary, "RhFindBlob")]
         internal static unsafe extern bool RhFindBlob(IntPtr hOsModule, uint blobId, byte** ppbBlob, uint* pcbBlob);
 
+#if CORERT
+        internal static uint RhGetLoadedModules(IntPtr[] resultArray)
+        {
+            // CORERT-TODO: This got broken within the last week or so. We don't care for single file.
+            if (resultArray != null)
```

Comment:
@nattress is it possible that the multifile changes broke `RhGetLoadedModules` and `RhGetModuleFromEEType`? I'm pretty sure the first one used to work.